tests/testStatsdClient.cpp

```diff
@@ -34,6 +65,7 @@ int main(int argc, char** argv) {
     }
 
     // Adjusts "toto" by +3
+    client.seed(19); // this seed gets a hit on the first call
```

Comment:
for messages sent with sampling set to less than `1.f` we modify the RNG's seed to make sure it will be sent